internal/cli/cmd/auth/login.go

```diff
@@ -55,6 +55,20 @@ func NewLoginCmd() *cobra.Command {
 				return err
 			}
 
+			userToken, err := auth.GenerateTokenFromUserAuth(ctx, userAuth)
```

Comment:
If this fails, we can have an old token cached and a new user.
Let's wipe the cached user + token at the beginning of `ns login`

Reply:
Do you prefer to remove files or store empty objects?

Reply:
Deleting the files feels very clean to me.

Reply:
Done